packages/vm/src/runTx.ts

```diff
@@ -267,6 +267,9 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
       // Note: in case we create a contract, we do this in EVMs `_executeCreate` (this is also correct in inner calls, per the EIP)
       state.addWarmedAddress(tx.to.buf)
     }
+    if (this._common.isActivatedEIP(3651)) {
+      state.addWarmedAddress(block.header.coinbase.buf)
+    }
```

Comment:
I think the reasoning why this was added to `runTx` in the first place is that it also produces the correct results in case you run a single tx. Note that this is in the same logic block as where we add the precompiles (these are always warm).

Reply:
Yes, I thought so, makes sense.